spec/models/user_spec.rb

```diff
@@ -454,17 +454,6 @@
     end
   end
 
-  it "#teams and #current_teams return unique team names" do
```

Comment:
Again removing it because now there is no teams to replace. Also thought of having similar test in roles, but there is no use-case of getting current groups in roles, only current roles are fetched and there it's supposed to return all roles of a group. So replacement may also be not possible.